.github/workflows/KaMPKit-Shared.yml

```diff
@@ -0,0 +1,27 @@
+name: KaMPKit-Shared
```

Comment:
Can we also add a readme badge for this flow?

Reply:
Done.